Source/WebCore/loader/DocumentLoader.cpp

```diff
@@ -2105,6 +2105,11 @@ void DocumentLoader::startLoadingMainResource()
 
     Ref<DocumentLoader> protectedThis(*this);
 
+    if (m_request.url().protocolIsAbout() && !(m_request.url().isAboutBlank() || m_request.url().isAboutSrcDoc())) {
+        cancelMainResourceLoad(frameLoader()->client().cannotShowURLError(m_request));
+        return;
+    }
```

Comment:
@cdumez @brentfulgham will it have any consequences on 

https://github.com/WebKit/WebKit/blob/0b71df88d47241c1024084a8f4edb0410d847520/Source/WebCore/page/Quirks.cpp#L822-L845

The origin bug was
https://bugs.webkit.org/show_bug.cgi?id=199933

which was about dealing with `about:invalid#zClosureZ`.
I don't need if the Quirk is needed. I haven't checked yet.

But would this make the Quirk useless?

Reply:
My patch doesn't have consequences on this quirk because this quirk gets applied *before* my new check.

So no, it wouldn't make this quirk useless. If anything, it would make the quirk more important because `about:invalid#zClosureZ` (which would get blocked by my patch) would get converted to `about:blank` (which is allowed to load).

Reply:
thanks for the clarification.